Tools-Override/tests.targets

```diff
@@ -55,11 +61,71 @@
 
     <TestCommandLine Condition="'$(Performance)'!='true'">$(TestProgram) $(TestArguments) {XunitTraitOptions}</TestCommandLine>
   </PropertyGroup>
+  
+  <!-- This will generate a version of the dotnet cli that runs with the live
+       build of corefx using the other components gathered from the stable 
+       version consumed from the Tools directory
+       To debug any issues set COREHOST_TRACE=1 and rerun dotnet
+  -->
+  <UsingTask TaskName="GenerateDepsJson" AssemblyFile="$(BuildToolsTaskDir)CoreFx.Tools.dll"/>
+  <PropertyGroup>
+    <NewSharedFxVersion>9.9.9</NewSharedFxVersion>
+    <TestDotNetCliPath>$(ToolsDir)dotnetcli\shared\Microsoft.NETCore.App\$(NewSharedFxVersion)</TestDotNetCliPath>
+    <OldDotNetCliPath>$(ToolsDir)dotnetcli</OldDotNetCliPath>
+  </PropertyGroup>
+  <Target Name="ConstructTestingHost" Condition="!Exists('$(TestDotNetCliPath)')" >
+    <ItemGroup>
+      <LiveDotNetCliComponents Include="$(RuntimePath)\*.*" Exclude="$(RuntimePath)\xunit.console.netcore.exe" />
+      <DotNetCliSharedFrameworkSdkRuntimeConfig Include="$(OldDotNetCliPath)\sdk\**\dotnet.runtimeconfig.json" />
+      <DotNetCliHostPolicy Include="$(OldDotNetCliPath)\shared\**\hostpolicy.dll" />
```

Comment:
My question was more along the lines of what happens if we end up finding more than one? I would prefer that we tie this to the 1.0.0 version using the property you will do for the replacement.